sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -264,7 +264,7 @@ String text = "Old Faithful is a geyser at Yellowstone Park.";
 for (LinkedEntity linkedEntity : textAnalyticsClient.recognizeLinkedEntities(text)) {
     System.out.println("Linked Entities:");
     System.out.printf("Name: %s, ID: %s, URL: %s, data source: %s.%n",
-        linkedEntity.getName(), linkedEntity.getId(), linkedEntity.getUrl(), linkedEntity.getDataSource());
+        linkedEntity.getName(), linkedEntity.getDataSourceLinkId(), linkedEntity.getUrl(), linkedEntity.getDataSource());
```

Comment:
why `getDataSourceLinkId` and not `getDataSourceEntitiesId` like the property name?

Reply:
will change to `getDataSourceEntitiesId`.